apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -158,6 +184,73 @@ const downloadButtonStyle = {
   textAlign: 'center'
 };
 
+class MicrobitDescription extends React.Component {
```

Comment:
i know you're following the conventions of this file, but these new components should be functional components (class components aren't necessary anymore). so the change looks something like...

```javascript
class MyComponent extends React.Component {
  render() {
    return <div>my component</div>;
  }
}

// instead becomes...
function MyComponent() {
  return <div>my component</div>;
}
```

here's an example of an existing functional component: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/CsaSkinnyBanner.jsx

Reply:
Easy fix.
I would love to hear more about the reasons behind this change.

Reply:
[this article](https://medium.com/geekculture/is-there-any-reason-to-still-use-react-class-components-9b6a1e6aa9ef) has some good historical context, but the basic gist is that functional components used to have limited features and now they're fully featured, so there's really no reason to use class components anymore!